### DISCUSSION TITLE
Separate animation actions for each instace of a model

### DISCUSSION THREAD

**Discussion by dulre159** [pmndrs/gltfjsx#108] [Q&A] [answered]
Hi i was wondring if it is possibile to separate the animation actions of each instace of the model?

For example i have a model which i want to replicate 4 times but i want to play an action only on one of them.

When i try to play an action on one instance al the other instances gets animated too :(.

How can i do this without having to load all the models seperatly from diffrent gltf files?
Thank you :)

**Comment by drcmda** [comment, marked as answer]
animations can't be re-used in threejs, they need to be cloned, but then it works: https://codesandbox.io/s/gltf-animations-re-used-k8phr

**Reply by dulre159** [reply]
After loading 16 models and animating them using this i am having performace issues... what i can do? :(

**Reply by dulre159** [reply]
Ok i think the performance problem was related to having the transparent property of the materials set to true
